Automattic/newspack-plugin
# Chore / IA Refactor Admin Tabs to Header

### All Submissions:


### Changes proposed in this Pull Request:

Refactored trait `Admin_Tabs` to `Admin_Header` so that the header can be used without requiring tabs.  Also refactored the only file that was using this code ("Advertising > Sponsors") to use the new code.  (The pure React pages seem to use a different header completely).

### How to test the changes in this Pull Request:

1.  Pull this branch on top of the `epic/ia` branch.
2.  In your browser, load WP Admin > Advertising > Sponsors page.
3.  Verify the header with tabs displays correctly with `margin-left: -20px` too.  (Attached)

![pr-admin-header-1](https://github.com/user-attachments/assets/d7bcc19e-b640-43e7-872d-f60f1c9f98b8)


### Other information:
